src/jaxsim/api/ode_data.py

```diff
@@ -165,8 +167,11 @@ def build_from_jaxsim_model(
 
         # Get the contact model from the `JaxSimModel`.
         match model.contact_model:
+
```

Comment:
This is a bit out of scope, but what about using `ruff format` instead of `black`, so we can get coherent spaces across the codesource?

Reply:
Mmh how does ruff format play with VSCode? AFAIK the default formatter is black. I need to double check in PyCharm.

Reply:
It's only about installing the `ruff` extension and choosing it from the dropdown menu
 
![image](https://github.com/user-attachments/assets/9eecfba4-5bdd-4eb8-a30a-879ef25cca69)

Reply:
It may be useful to checkout https://docs.astral.sh/ruff/formatter/black/

Reply:
Ok, we can discuss this f2f next week before starting with the possible migration.